### ISSUE TITLE
Cross-compile for Scala.js (using JS's JSON instead of Jackson)

### ISSUE THREAD

**Issue by rpiaggio** [playframework/play-json#29]

**Comment by cchantep** [comment]
Duplicate of #13 

> Please first check the existing ticket.

**Comment by rpiaggio** [comment]
@cchantep This is great.

Sorry, I checked existing issues, but not PRs. I usually create an issue before starting to work on something so that I can cross-reference it from commit messages, but I understand that no everyone works this way.

I will fork from your repo and give it a try.
